packages/opencensus-core/src/index.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+import * as logger from './common/console-logger';
+import * as classes from './index-classes';
+import * as types from './index-types';
+
+export {classes, logger, types};
```

Comment:
This shows up as a necessity to avoid name collision between interfaces and implementations, as we agree to avoid using Impls suffixes for implementations and decided to use module namespace to distinguish between interfaces from implementations.